tools/docker/Dockerfile

```diff
@@ -52,29 +52,28 @@ ENV PATH="/root/cmake-3.16.3-Linux-x86_64/bin:${PATH}"
 
 # Configuration
 ## Maven Proxy - required for usage from behind a corporate proxy. Corporate proxy should be bound to the docker network bridge at 172.17.0.1.
-## If you are not behind a corporate proxy this can be ignored and shouldn't interfere with regular usage.
-RUN mkdir -p /root/.m2
-RUN echo '\n\
-<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" \n\
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" \n\
-  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 \n\
-                  http://maven.apache.org/xsd/settings-1.0.0.xsd"> \n\
-  <proxies> \n\
-    <proxy> \n\
-      <active>true</active> \n\
-      <protocol>http</protocol> \n\
-      <host>172.17.0.1</host> \n\
-      <port>3128</port> \n\
-      <nonProxyHosts>127.0.0.1</nonProxyHosts> \n\
-    </proxy> \n\
-  </proxies> \n\
-</settings> \n'\
->> /root/.m2/settings.xml
+#RUN mkdir -p /root/.m2
+#RUN echo '\n\
+#<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" \n\
+  #xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" \n\
+  #xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 \n\
+                  #http://maven.apache.org/xsd/settings-1.0.0.xsd"> \n\
+  #<proxies> \n\
+    #<proxy> \n\
+      #<active>true</active> \n\
+      #<protocol>http</protocol> \n\
+      #<host>172.17.0.1</host> \n\
+      #<port>3128</port> \n\
+      #<nonProxyHosts>127.0.0.1</nonProxyHosts> \n\
+    #</proxy> \n\
+  #</proxies> \n\
+#</settings> \n'\
+#>> /root/.m2/settings.xml
```

Comment:
can this be removed?

Reply:
I would leave that here commented for the guys who are sitting behind a corporate proxy